devel/bazel/Portfile

```diff
@@ -72,6 +72,12 @@ build {
     # Put executable in main working folder
     xinstall -m 755 -W ${worksrcpath}/output ${name} ${worksrcpath}
 
+    # Get ready to build the bash completion script
+    file delete -force /var/tmp/_${name}_root
+    exec -ignorestderr -- git init ${worksrcpath}
+    exec -ignorestderr -- git add ${worksrcpath}
+    exec -ignorestderr -- git commit -m GitRepo
```

Comment:
Why do you need to make a dummy commit during the build phase?

It also looks a bit suspicious that a build would fail if you don't remove a file.

Reply:
Oddly enough, I can't build the bash completion script without deleting that temporary directory and having a local git repo in that state. I suspect it's a bug that popped up after 0.5.2. So essentially these changes work around that problem.

Reply:
Never use `exec`, only `system`.

I don't understand why this would be necessary, though. If a git repo is assumed in any build rule, better patch that.

Reply:
Why only `system` rather than `exec`? Also, could you point me to a good document for `system`? I'm not finding one online.

Reply:
From `man portfile`:

     system [-W dir] commandline
         Execute a program. See system(3).  For calls to install(1) please use xinstall.  For calls to mv(1), cp(1), rm(1) or
         similar, please use the built-in commands or file if they don't meet your requirements.

         -W      Change to dir before working.

Reply:
Without that bit of code, the bazel build errors out like this
```
Build successful! Binary is here: /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_bazel/bazel/work/bazel-0.9.0-dist/output/bazel
xinstall: chdir(/opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_bazel/bazel/work/bazel-0.9.0-dist/output)
xinstall: bazel -> /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_bazel/bazel/work/bazel-0.9.0-dist/bazel
Extracting Bazel installation...
.....................
Loading: 
Loading: 0 packages loaded
Analyzing: target //scripts:bazel-complete.bash (3 packages loaded)
Analyzing: target //scripts:bazel-complete.bash (4 packages loaded)
Analyzing: target //scripts:bazel-complete.bash (73 packages loaded)
Analyzing: target //scripts:bazel-complete.bash (75 packages loaded)
Analyzing: target //scripts:bazel-complete.bash (195 packages loaded)
INFO: Analysed target //scripts:bazel-complete.bash (195 packages loaded).
INFO: Found 1 target...
[0 / 2] [-----] BazelWorkspaceStatusAction stable-status.txt
ERROR: Process exited with status 128: Process exited with status 128
fatal: Not a git repository (or any of the parent directories): .git
Target //scripts:bazel-complete.bash failed to build
INFO: Elapsed time: 25.182s, Critical Path: 0.14s
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
Error: Failed to build bazel: child process exited abnormally
Error: See /opt/local/var/macports/logs/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_devel_bazel/bazel/main.log for details.
Error: Follow https://guide.macports.org/#project.tickets to report a bug.
Error: Processing of port bazel failed
```

Reply:
Anyway, with that bit of code in, 0.9.0 builds without intervention
```
$ port -v installed bazel
The following ports are currently installed:
  bazel @0.9.0_0 (active) platform='darwin 17' archs='x86_64' date='2017-12-30T17:32:13-0800'
```
Thanks!